### DISCUSSION TITLE
Anti Spam Details?

### DISCUSSION THREAD

**Discussion by brugmodius** [KadeArchive/Kade-Engine#32] [Q&A]
I know that an anti spam measure is in place, but how does it actually work? I randomly die midway through some modded songs and I feel like it's tied to the anti spam, as I'm hitting the notes.

**Comment by Assestions** [comment]
if this is for a older version of kade engine please get 1.4, it fixed like evrything except for freeplay

**Reply by Nandovsk** [reply]
> se for para uma versão mais antiga do motor kade, obtenha 1.4, ele é consertado como tudo, exceto para freeplay

Whitty's mod still has bad anti-spam
